test/behavior/packed_struct_equality.zig

```diff
@@ -0,0 +1,22 @@
+const std = @import("std");
+const expect = std.testing.expect;
+
+test "packed struct equality" {
+    const Foo = packed struct {
+        a: u4,
+        b: u4,
+    };
+
+    const S = struct {
+        fn doTest(x: Foo, y: Foo) !void {
+            try expect(x == y);
+            try expect(!(x != y));
+        }
+    };
+
+    const x: S = .{ .a = 1, .b = 2 };
+    const y: S = .{ .b = 2, .a = 1 };
```

Comment:
x and y are of type S, but S has no fields?

Reply:
Yeah -- @Rexicon226, you forgot to import this new file into `behavior.zig`. I'd suggest just putting it into an existing file, like `structs.zig` (`struct.zig`? can't check rn)

Reply:
Or `packed-struct.zig`.

Reply:
Thanks for noticing, I always forget that the behavior tests aren't auto-gathered like the compile error ones, moved to `packed-struct.zig`.